pkg/xen/Dockerfile

```diff
@@ -19,8 +19,6 @@ RUN [ -f "$(basename ${XEN_SOURCE})" ] || tar --absolute-names -xz < /xen.tar.gz
 
 WORKDIR /xen/xen
 COPY *.patch arch /tmp/
-RUN cp /tmp/"$(uname -m)"/*.patch /tmp/
-RUN for p in /tmp/*.patch ; do patch -p1 < "$p" || exit 1 ; done
```

Comment:
@roja-zededa , under the pkg/xen folder we have all these patch files:

```
pkg/xen
├── 0001-disable-Werror-to-build-under-gcc-11.2.patch
└── arch
    ├── aarch64
    │   ├── 0002-arm-efi-mem-detection.patch
    │   ├── 0003-arch-arm-small-hack-for-rpi4-usb.patch
    │   ├── 0004-no-warn.patch
    │   └── 0005-smbios.patch
    └── x86_64
        ├── 0001-gettime-considered-harmful.patch
        └── 0002-xen-spoofing.patch
```
my point is that if you are not applying these patches anymore, they should be removed (the files), with `git rm <patch file>`

then you should also remove the line 21 (`COPY *.patch arch /tmp/`) since it will be not needed anymore....

Reply:
Ideally this removal should happen in a separate commit.

Reply:
I think `git mv` would be more useful, I just tried it out locally:
```
 rename pkg/xen-tools/{patches-4.15.0 => patches-4.19.0}/00-musl-support.patch (100%)
 rename pkg/xen-tools/{patches-4.15.0 => patches-4.19.0}/01-enable-gmp-patching.patch (100%)
 rename pkg/xen-tools/{patches-4.15.0 => patches-4.19.0}/04-disable-vif-allow-all-iptables.patch (100%)
 rename pkg/xen-tools/{patches-4.15.0 => patches-4.19.0}/08-memlogd-logger.patch (87%)
 rename pkg/xen-tools/{patches-4.15.0 => patches-4.19.0}/09-enable-kvm.patch (100%)
 rename pkg/xen-tools/{patches-4.15.0 => patches-4.19.0}/10-bridge-helper-support.patch (61%)
 rename pkg/xen-tools/{patches-4.15.0 => patches-4.19.0}/11-char-socket-revert.patch (100%)
 rename pkg/xen-tools/{patches-4.15.0 => patches-4.19.0}/12-disable-Werror-to-build-under-gcc-11.2.patch (100%)
 rename pkg/xen-tools/{patches-4.15.0 => patches-4.19.0}/13-qemu-Add-cpu-pin-and-cpumask-options.patch (89%)
 rename pkg/xen-tools/{patches-4.15.0 => patches-4.19.0}/14-qemu-Init-CPU-mask-per-VCPU.patch (76%)
 rename pkg/xen-tools/{patches-4.15.0 => patches-4.19.0}/15-qemu-Set-the-affinity-of-QEMU-threads-according-to-t.patch (100%)
 rename pkg/xen-tools/{patches-4.15.0 => patches-4.19.0}/x86_64/05-xen-spoofing.patch (89%)
 rename pkg/xen-tools/{patches-4.15.0 => patches-4.19.0}/x86_64/07-patch-seabios.patch (68%)
```

the patches are not the same as before

Reply:
@christoph-zededa Done!